src/components/Trending/Trending.js

```diff
@@ -2,30 +2,86 @@ import DataEvaluationContext from "../../contexts/DataEvaluationContext";
 import UserContext from "../../contexts/UserContext";
 import { TextWithHighlightedHashtags } from "../../utils/TextAdjustmentsUtils";
 import { getTrendingTopics } from "../../service/service";
+import { sendAlert } from "../../utils/helpers/Alerts";
 
 import { useContext, useEffect, useState } from "react";
 import styled from "styled-components";
+import { useHistory } from "react-router";
 
 export default function Trending() {
     const [trendingTopics, setTrendingTopics ] = useState([]);
     const { login } = useContext(UserContext);
     const { isDataBeingEvaluated } = useContext(DataEvaluationContext);
+    const [searchedHashtag, setSearchedHashtag] = useState("");
+    const history = useHistory();
     
     useEffect(() => {
         if(login.token) {
             getTrendingTopics(login.token, setTrendingTopics)
         }
     },[login,isDataBeingEvaluated]);
+
+    function searchHashtag(e) {
+        if(e.keyCode !== 13) return;
+
+        if(searchedHashtag.includes(" ")) {
+            return sendAlert("error", "Uma hashtag não pode ter espaços em branco!");
+        }
+        if(!searchedHashtag) {
+            return sendAlert("error", "Digite alguma hashtag para fazer sua busca!");
+        }
+        history.push(`/hashtag/${searchedHashtag}`);
+    }
```

Comment:
I'd suggest to include a .toLowerCase there, since the API only sends low case hashtags to trendings.